server/superdesk/upload.py

```
@@ -37,11 +37,14 @@ def get_upload_as_data_uri(media_id):
             data,
             mimetype=media_file.content_type,
             direct_passthrough=True)
-        response.content_length = media_file.length
-        response.last_modified = media_file.upload_date
-        response.set_etag(media_file.md5)
-        response.cache_control.max_age = cache_for
-        response.cache_control.s_max_age = cache_for
+        try:
+            response.content_length = media_file.length
+            response.last_modified = media_file.upload_date
+            response.set_etag(media_file.md5)
+            response.cache_control.max_age = cache_for
+            response.cache_control.s_max_age = cache_for
+        except Exception:
+            pass
```

Do we really want to silence the exception?